ccd-definition/CaseField/CareSupervision/caseField.json

```diff
@@ -1303,5 +1303,33 @@
     "FieldType": "Label",
     "SecurityClassification": "Public",
     "Searchable": "N"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "ID": "uploadApplicationDocumentsLabel",
+    "Label": "Select the supporting or annex documents you're sending with this application. <br> These can be part of the social work evidence template (SWET) or separate.",
```

Comment:
technically we don't need to do this label in this PR right? I think PR should just be for the fields that need to be present as soon as we merge (i.e new fields that are being populated in old events). I guess the R permissions for other users are also not needed if you wanted PR to be smaller, but no harm in putting them

Reply:
removed